bin/run_brick.rb

```diff
@@ -14,6 +14,27 @@ def get_brick_params(prefix)
   ENV.select { |k,| k.to_s.match(/^#{prefix}.*/) }.map { |k, v| [k.slice(prefix.length..-1), v] }.to_h
 end
 
+def log_execution_result(params, log, brick_type, error)
```

Comment:
log_execution_result --> log_execution_summary_error

Reply:
Done fix it.